### PR TITLE
[15.0] [MIG] account_payment_mode

### PR DESCRIPTION
**Pull request by misern2**
Standrad Migration

### CODE REVIEW

**Review by ajaniszewska-dev — APPROVED** [review]

File: account_payment_mode/models/account_journal.py [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+        all_out = self.env["account.payment.method"].search(
+            [("payment_type", "=", "outbound")]
+        )
+        return all_out
+
+    def _default_inbound_payment_methods(self):
+        all_in = self.env["account.payment.method"].search(
+            [("payment_type", "=", "inbound")]
+        )
+        return all_in
```
Comment (i-vyshnevska) [inline account_payment_mode/models/account_journal.py:22]: fields to this method were removed 
do we need to migrate them or remove them as well?

**Review by i-vyshnevska — COMMENTED** [review]
translation commits can be squashed

File: account_payment_mode/models/account_journal.py (hunk 1 above)
Comment (gurneyalex) [reply account_payment_mode/models/account_journal.py:22]: The methods are still present and used in account.journal:

https://github.com/odoo/odoo/blob/87e913b892c2518a716fceb247b57ab8c955a07f/addons/account/models/account_journal.py#L32-L36

https://github.com/odoo/odoo/blob/87e913b892c2518a716fceb247b57ab8c955a07f/addons/account/models/account_journal.py#L280-L302

so it totally makes sense to keep them in the module. 

(actually these were never method to compute a default value for a field on account.journal, but used to compute a default value for a payment being recorded on that journal)

File: account_payment_mode/tests/test_account_payment_mode.py [hunk 2]
```diff
@@ -0,0 +1,129 @@
+# Copyright 2016-2020 ForgeFlow S.L.
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl.html).
+
+from odoo.exceptions import UserError, ValidationError
+from odoo.tests.common import TransactionCase
+
+
+class TestAccountPaymentMode(TransactionCase):
+    def setUp(self):
```
Comment (simahawk) [inline account_payment_mode/tests/test_account_payment_mode.py]: pls move this setup code to `setUpClass` and start like this:

```py
    @classmethod
    def setUpClass(cls):
        super().setUpClass()
        cls.env = cls.env(context=dict(cls.env.context, tracking_disable=True))
```

File: account_payment_mode/views/account_payment_method.xml [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+                </sheet>
+            </form>
+        </field>
+    </record>
+    <record id="account_payment_method_tree" model="ir.ui.view">
+        <field name="name">account_payment_method.tree</field>
+        <field name="model">account.payment.method</field>
+        <field name="arch" type="xml">
+            <tree>
+               <!--<field name="sequence" widget="handle" /> -->
```
Comment (simahawk) [inline account_payment_mode/views/account_payment_method.xml]: needed?

File: account_payment_mode/models/account_journal.py (hunk 1 above)
Comment (simahawk) [reply account_payment_mode/models/account_journal.py:22]: @i-vyshnevska fine for you?

**Review by simahawk — CHANGES_REQUESTED** [review]

File: account_payment_mode/tests/test_account_payment_mode.py (hunk 2 above)
Comment (misern2) [reply account_payment_mode/tests/test_account_payment_mode.py]: :+1:

File: account_payment_mode/views/account_payment_method.xml (hunk 3 above)
Comment (misern2) [reply account_payment_mode/views/account_payment_method.xml]: Not needed. Removed. :+1:

**Review by i-vyshnevska — APPROVED** [review]

**Review by simahawk — APPROVED** [review]

**Comment by pedrobaeza** [comment]
@simahawk FYI we have that new command that I have put in the previous comment for auto-filling migration issue and milestone.

**Comment by simahawk** [comment]
> @simahawk FYI we have that new command that I have put in the previous comment for auto-filling migration issue and milestone.

waaaat? I missed it! Cool! :smile: 
Thanks for the heads up :+1:

**Pull request merged**